.github/CODEOWNERS

```diff
@@ -13,9 +13,7 @@
 /CODE_OF_CONDUCT.md @nodejs/tsc
 /CONTRIBUTING.md @nodejs/tsc
 /LICENSE @nodejs/tsc
-/doc/guides/contributing/*.md @nodejs/tsc
-/doc/guides/collaborator-guide.md @nodejs/tsc
-/doc/guides/offboarding.md @nodejs/tsc
+/doc/contributing/*.md @nodejs/tsc
```

Comment:
I think this means that the TSC will be pinged for changes to any of the guides now.

Reply:
Thanks, good idea

Reply:
It was more of a comment/heads up. The previous directory structure and config only pinged TSC on a subset of documents.

Reply:
If that becomes a problem I'm happy to change in later on.